include.js

```diff
@@ -27,20 +34,6 @@ let cards = [
     author: 'stormworm9',
     githubLink: 'https://github.com/stormworm9'
   },
-  {
-    artName: 'Loading Page',
-    pageLink: './Art/SaumyaKumar-09/Animation%204/index.html',
-    imageLink: './Art/SaumyaKumar-09/Animation%204/Animation4.gif',
-    author: 'SaumyaKumar-09',
-    githubLink: 'https://github.com/SaumyaKumar-09'
-  },
-  {
-    artName: 'Windows start animation',
-    pageLink: './Art/SaumyaKumar-09/Animation%202/index.html',
-    imageLink: './Art/SaumyaKumar-09/Animation%202/Animation2.gif',
-    author: 'SaumyaKumar-09',
-    githubLink: 'https://github.com/SaumyaKumar-09'
-  },
```

Comment:
# Feedback -  wrong folder naming
- please remove any space in folder naming ( all animation folder ) and use hyphens or underscore instead

Reply:
Okay. I changed as you requested.